From 82d94f9e070588af81109bd9eeb25fe9b2a69acb Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Thu, 1 Aug 2019 20:12:38 +0800
Subject: [PATCH] Update shared.yml in accounts_passwords_pam_tally2_deny and
 rule.yml in cracklib_accounts_password_pam_retry

Update the content of shared.yml in rule accounts_passwords_pam_tally2_deny and
 rule.yml in rule cracklib_accounts_password_pam_retry.

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 .../ansible/shared.yml                             | 30 ++++----
 .../ansible/shared.yml                             | 81 ----------------------
 .../cracklib_accounts_password_pam_retry/rule.yml  |  4 +-
 3 files changed, 17 insertions(+), 98 deletions(-)
 delete mode 100644 linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/ansible/shared.yml

diff --git a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny/ansible/shared.yml b/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny/ansible/shared.yml
index d2b08c0..2d5ffbb 100644
--- a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny/ansible/shared.yml
+++ b/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny/ansible/shared.yml
@@ -3,9 +3,9 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- (xccdf-var var_accounts_passwords_pam_faillock_deny)
+- (xccdf-var var_accounts_passwords_pam_tally2_deny)
 
-- name: Add auth pam_faillock preauth deny before pam_unix.so
+- name: Add auth pam_tally2 preauth deny before pam_unix.so
   pamd:
     name: "{{ item }}"
     type: auth
@@ -13,30 +13,30 @@
     module_path: pam_unix.so
     new_type: auth
     new_control: required
-    new_module_path: pam_faillock.so
+    new_module_path: pam_tally2.so
     module_arguments: 'preauth
         silent
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
+        deny={{ var_accounts_passwords_pam_tally2_deny }}'
     state: before
   loop:
     - system-auth
     - password-auth
 
-- name: Add deny argument to auth pam_faillock preauth
+- name: Add deny argument to auth pam_tally2 preauth
   pamd:
     name: "{{ item }}"
     type: auth
     control: required
-    module_path: pam_faillock.so
+    module_path: pam_tally2.so
     module_arguments: 'preauth
         silent
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
+        deny={{ var_accounts_passwords_pam_tally2_deny }}'
     state: args_present
   loop:
     - system-auth
     - password-auth
 
-- name: Add auth pam_faillock authfail deny after pam_unix.so
+- name: Add auth pam_tally2 authfail deny after pam_unix.so
   pamd:
     name: "{{ item }}"
     type: auth
@@ -44,29 +44,29 @@
     module_path: pam_unix.so
     new_type: auth
     new_control: '[default=die]'
-    new_module_path: pam_faillock.so
+    new_module_path: pam_tally2.so
     module_arguments: 'authfail
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
+        deny={{ var_accounts_passwords_pam_tally2_deny }}'
     state: after
   loop:
     - system-auth
     - password-auth
 
-- name: Add deny argument to auth pam_faillock authfail
+- name: Add deny argument to auth pam_tally2 authfail
   pamd:
     name: "{{ item }}"
     type: auth
     new_type: auth
     control: '[default=die]'
-    module_path: pam_faillock.so
+    module_path: pam_tally2.so
     module_arguments: 'authfail
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
+        deny={{ var_accounts_passwords_pam_tally2_deny }}'
     state: args_present
   loop:
     - system-auth
     - password-auth
 
-- name: Add account pam_faillock before pam_unix.so
+- name: Add account pam_tally2 before pam_unix.so
   pamd:
     name: "{{ item }}"
     type: account
@@ -74,7 +74,7 @@
     module_path: pam_unix.so
     new_type: account
     new_control: required
-    new_module_path: pam_faillock.so
+    new_module_path: pam_tally2.so
     state: before
   loop:
     - system-auth
diff --git a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/ansible/shared.yml b/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/ansible/shared.yml
deleted file mode 100644
index d2b08c0..0000000
--- a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/ansible/shared.yml
+++ /dev/null
@@ -1,81 +0,0 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
-# reboot = false
-# strategy = restrict
-# complexity = low
-# disruption = low
-- (xccdf-var var_accounts_passwords_pam_faillock_deny)
-
-- name: Add auth pam_faillock preauth deny before pam_unix.so
-  pamd:
-    name: "{{ item }}"
-    type: auth
-    control: sufficient
-    module_path: pam_unix.so
-    new_type: auth
-    new_control: required
-    new_module_path: pam_faillock.so
-    module_arguments: 'preauth
-        silent
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
-    state: before
-  loop:
-    - system-auth
-    - password-auth
-
-- name: Add deny argument to auth pam_faillock preauth
-  pamd:
-    name: "{{ item }}"
-    type: auth
-    control: required
-    module_path: pam_faillock.so
-    module_arguments: 'preauth
-        silent
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
-    state: args_present
-  loop:
-    - system-auth
-    - password-auth
-
-- name: Add auth pam_faillock authfail deny after pam_unix.so
-  pamd:
-    name: "{{ item }}"
-    type: auth
-    control: sufficient
-    module_path: pam_unix.so
-    new_type: auth
-    new_control: '[default=die]'
-    new_module_path: pam_faillock.so
-    module_arguments: 'authfail
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
-    state: after
-  loop:
-    - system-auth
-    - password-auth
-
-- name: Add deny argument to auth pam_faillock authfail
-  pamd:
-    name: "{{ item }}"
-    type: auth
-    new_type: auth
-    control: '[default=die]'
-    module_path: pam_faillock.so
-    module_arguments: 'authfail
-        deny={{ var_accounts_passwords_pam_faillock_deny }}'
-    state: args_present
-  loop:
-    - system-auth
-    - password-auth
-
-- name: Add account pam_faillock before pam_unix.so
-  pamd:
-    name: "{{ item }}"
-    type: account
-    control: required
-    module_path: pam_unix.so
-    new_type: account
-    new_control: required
-    new_module_path: pam_faillock.so
-    state: before
-  loop:
-    - system-auth
-    - password-auth
diff --git a/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pamcracklib/cracklib_accounts_password_pam_retry/rule.yml b/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pamcracklib/cracklib_accounts_password_pam_retry/rule.yml
index e3a9dcb..f6b70de 100644
--- a/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pamcracklib/cracklib_accounts_password_pam_retry/rule.yml
+++ b/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pamcracklib/cracklib_accounts_password_pam_retry/rule.yml
@@ -6,7 +6,7 @@ description: |-
     To configure the number of retry prompts that are permitted per-session:
     <br /><br />
     Edit the <tt>pam_cracklib.so</tt> statement in
-    <tt>/etc/pam.d/system-auth</tt> to show
+    <tt>/etc/pam.d/common-password</tt> to show
     <tt>retry=<sub idref="var_password_pam_retry" /></tt>, or a lower value
     if site policy is more restrictive.
     <br /><br />
@@ -37,7 +37,7 @@ ocil_clause: 'it is not the required value'
 
 ocil: |-
     To check how many retry attempts are permitted on a per-session basis, run the following command:
-    <pre>$ grep pam_cracklib /etc/pam.d/system-auth</pre>
+    <pre>$ grep pam_cracklib /etc/pam.d/common-password</pre>
     The <tt>retry</tt> parameter will indicate how many attempts are permitted.
     The DoD required value is less than or equal to 3.
     This would appear as <tt>retry=3</tt>, or a lower value.
-- 
1.9.1

